mkdocs/pages/en/devbook/accounts/account-metadata.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Account Metadata
+---
+
+# Adding Metadata to an Account
+
+<Accounts:> can store <metadata:> as key-value pairs.
+
+This tutorial shows how to add metadata to an account, retrieve metadata from the network, and update existing values.
+
+In this example, an account adds a username to itself and then modifies it.
+
+## Prerequisites
+
+Before you start, make sure to:
+
+- Set up your development environment.
+  See [Setting Up a Development Environment](../start/setup.md).
+- Create an <account:> to add metadata to, either
+  [from code](./create-from-private-key.md) or
+  [by using a wallet](../../userbook/wallet/create-account.md).
+- Obtain <XYM:> to pay for the transaction fee.
+  See [Getting Testnet Funds from the Faucet](./testnet-faucet.md).
+
+Additionally, review the [Transfer transaction](../transactions/transfer.md) tutorial to understand how
+transactions are announced and confirmed, and the
+[Complete Aggregate transaction](../transactions/complete-aggregate.md) tutorial to understand how aggregate
+transactions work.
+
+## Full Code
+
+{% import 'tutorial.jinja2' as tutorial with context %}
+
+{{ tutorial.code_full('devbook/transactions/account-metadata', ['py', 'js']) }}
+
+## Code Explanation
+
+This tutorial demonstrates adding new metadata to an account and then updating that metadata.
+
+### Setting Up the Account
+
+{{ tutorial.code_snippet(['py:53:61', 'js:51:59']) }}
+
+The snippet reads the signer's private key from the `SIGNER_PRIVATE_KEY` environment variable, which defaults to a
+test key if not set.
+The signer's address is derived from the public key.
+
+In this tutorial, the signer adds metadata to their own account.
+Adding metadata to a different account requires the target to cosign the transaction.
+
+### Fetching Network Time and Fees
+
+{{ tutorial.code_snippet(['py:64:82', 'js:62:80']) }}
+
+Network time and recommended fees are fetched from <get:/node/time> and <get:/network/fees/transaction> respectively, following the process described in the [Transfer Transaction](./transfer.md) tutorial.
+
+### Defining the Metadata
+
+{{ tutorial.code_snippet(['py:84:87', 'js:82:85']) }}
+
+Each metadata entry is identified by a **scoped metadata key**, a 64-bit value chosen by the metadata creator.
+The combination of the signer's address, the target account's address, and the scoped metadata key uniquely
+identifies the metadata entry.
+
+!!! note "Multiple entries with the same key"
+
+    Because the signer's address is part of the unique identifier, multiple accounts can add metadata with the same
+    scoped key to a target (e.g., Account A and Account B can add a different `username` to Account C).
+    Each entry remains separate and can only be updated by its signer.
```

Comment:
```suggestion
    Because the signer's address is part of the unique identifier, different accounts can use the same scoped metadata key 
    on the same target account without conflict.
    
    For example, Account A and Account B can both use the key `username` when adding metadata to Account C,
    resulting in two distinct metadata entries.

    Each entry is independent and can only be updated by the account that originally created it.
```